### ISSUE TITLE
add mavlink msg status text

### ISSUE THREAD

**Issue by Imen-D** [krzysztofkuczek/QCZEK_LRS#20]
Hi, sometime while we on the field we can't arm the plane (arduplane), so it will be help, if we can know the problem with malink status msg.

thanks

**Comment by ninja-zx11** [comment]
Yes please add STATUSTEXT.  Its really helpful to get the needed info before arming on an Ardupilot. Its gonna be a icing on the cake!!!
